pandas/io/formats/latex.py

```diff
@@ -236,3 +237,74 @@ def _print_cline(self, buf, i, icol):
                           .format(cl=cl[1], icol=icol))
         # remove entries that have been written to buffer
         self.clinebuf = [x for x in self.clinebuf if x[0] != i]
+
+    def _write_tabular_begin(self, buf, column_format):
+        """
```

Comment:
can you add Parameters to the doc-string; ideally type annotate the args as well